### PR TITLE
Fix tests of the Swagger schema of Google protobuf wrapper fields.

### PR DESCRIPTION
**Pull request by j6carey**
It seems that attempting to use conditional compilation on doctests
of the Swagger format for fields of Google protobuf wrapper type
silently suppresses those tests.

This commit replaces such doctests with HUnit tests.

### CODE REVIEW

File: tests/TestCodeGen.hs [hunk 1]
```diff
@@ -46,6 +49,60 @@ codeGenTests = testGroup "Code generator unit tests"
...
+           "{\"properties\":{\"wrapper\":{\"format\":\"byte\",\"type\":\"string\"}},\"type\":\"object\"}"
+    ]
+  where
+    prop ::
+      forall a .
+      (ToSchema a, Typeable a) =>
+      LBS.ByteString ->
+      LBS.ByteString ->
+      TestTree
+    prop wrapperFormat noWrapperFormat =
```
Comment (jcarr-awake) [inline tests/TestCodeGen.hs:89]: What about a slightly clearer name for this, expectSchema? Or something else? If not for the context I have this test would be very unclear

Comment (j6carey) [reply tests/TestCodeGen.hs:89]: Good point; how does 02b0f881f3b3a39b09f6896d7d9cdd9760921748 look?

Comment (jcarr-awake) [reply tests/TestCodeGen.hs:89]: Well you didn't change the definition :)

Comment (jcarr-awake) [reply tests/TestCodeGen.hs:89]: But should be good in that case

Comment (j6carey) [reply tests/TestCodeGen.hs]: Hopefully fixed by f3e273a55ff770e47bd4f62b1a42171ae037f162 .

**Review by jcarr-awake — APPROVED** [review]

File: tests/TestCodeGen.hs (hunk 1 above)
Comment (j6carey) [reply tests/TestCodeGen.hs]: CI passed.  Resolving this issue.

**Pull request merged**